tests/unittests/OperatorTest.cpp

```diff
@@ -3362,6 +3362,63 @@ TEST_P(InterpAndCPU, insertTensorTest) {
   }
 }
 
+/// Test RowwiseQuantizedFullyConnected Node.
+TEST_P(InterpOnly, rowwiseQuantizedFCTest) {
+  // In this test we subtract the outputs of a row-wise quantized FC and a
+  // floating-point FC and ensure that the error is below some low value.
+  auto *input = mod_.createVariable(ElemKind::FloatTy, {1, 300}, "in");
+  auto *fc = F_->createFullyConnected("FC", input, 30);
+  auto *res =
+      mod_.createVariable(ElemKind::FloatTy, fc->getResult().dims(), "res");
+
+  auto weights = fc->getWeights();
+  auto bias = fc->getBias();
+
+  input->getPayload().getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  llvm::cast<Variable>(bias)->getPayload().getHandle().randomize(
+      0, 0.00001, mod_.getPRNG());
+  llvm::cast<Variable>(weights)->getPayload().getHandle().randomize(
+      -1.1, 1.1, mod_.getPRNG());
+
+  // The FC fomula is I * W + B, while the RWQFC is I * transpose(W) + B.
+  // So get the tranpose of weights from the above FC.
+  auto *newWeights = mod_.createVariable(
+      ElemKind::FloatTy, {weights.dims()[1], weights.dims()[0]}, "newW");
+
+  newWeights->getPayload().transpose(
+      &(llvm::cast<Variable>(weights)->getPayload()), {1, 0});
+
+  TypeRef resTy = mod_.uniqueType(ElemKind::Int8QTy, res->dims(), 0.15, 4);
```

Comment:
can you comment on how quantization params are chosen?

Reply:
There is an unittest IntFC which tests the quantized FC. So I just copied the params from there...

Reply:
I think it'd be useful to at least specify the floating point ranges represented by these {s,o}